modules/update-using-custom-machine-config-pools-about.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * updating/update-using-custom-machine-config-pools.adoc
+
+[id="update-using-custom-machine-config-pools-about_{context}"]
+= About performing an IPSU
+
+This topic describes the general workflow of this in-place software update (IPSU) process.
```

Comment:
We should remove this  `in-place software update (IPSU) process` as explained previously.